content/articles/recover-deleted-files-with-foremost/index.md

```diff
@@ -0,0 +1,139 @@
+### **Recovering deleted files using foremost**
+
+Unintentional deletion of important files can be tragic. This has led to operating systems having a restore point. In Linux, the restore point is the trash. In Windows, the restore point is the recycle bin. Sometimes we delete these files in the restore points too. At this point, we opt to visit data recovery centers or buy expensive data recovery software.
+
+### Introduction
+
+[**Foremost**](https://pkgs.org/download/foremost) is a free data retrieval software. You can download it on the internet. It operates on Linux operating systems. Throughout this tutorial, we will be using Kali Linux. Anyone using any other Linux distribution can follow through.
+
+### What we will learn
+
+In this tutorial, we shall be looking at:
+- Installation of **foremost** in Kali Linux.
+- Tools included in the **foremost** package and their use.
+- Use of **foremost** in data recovery of lost/deleted files from a USB drive.
+- Recovering deleted files from a whole hard disk. 
+- Recovering deleted files from a particular partition of a hard drive.
+- Recovering files of all types.
+
```

Comment:
The format for writing your `Table of contents` section is wrong. Please follow the format in this [tutorial](https://github.com/section-engineering-education/engineering-education/pull/3310/files)